.github/workflows/build_wheels.yml

```diff
@@ -0,0 +1,104 @@
+name: PyPI
+
+on:
+  # Trigger on pull requests.
+  pull_request:
+
+  # Trigger on pushes to the mainline branches and version tags. This prevents building commits
+  # twice when the pull request source branch is in the same repository.
+  push:
+    branches:
+      - "maint"
+      - "master"
+    tags:
+      - "v*"
+
+  # Trigger on request.
+  workflow_dispatch:
+
+env:
+  SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, windows-2019, macos-10.15]
+
+    steps:
+      - uses: actions/checkout@v2.3.4
+
+      - name: Install Visual C++ for Python 2.7
```

Comment:
I think the Python 2.7 steps could be deleted.

Reply:
You are right.